Makefile

```diff
@@ -108,3 +108,12 @@ apidoc:
 apidoc_clean:
 	rm -rf apidoc
 endif
+
+## Editor navigation files
+C_SOURCE_FILES = $(wildcard include/*/*.h library/*.[hc] programs/*/*.[hc] tests/suites/*.function)
```

Comment:
Does this miss `wince_main.c`?

*wince*

Reply:
Indeed it does. But practically, `wince_main.c` does not declare any identifier that is used elsewhere in the code. And philosophically, `wince_main.c` is platform-specific and may potentially declare identifiers that clash with identifiers of the same name on “normal” platforms, so it shouldn't be indexed.

Reply:
OK